### ISSUE TITLE
Consistent methods of interacting with a list of options

### ISSUE THREAD

**Issue by crckhub** [IanMitchell/aquarius#845]
**Is this a Command (activated) or Plugin (passive)?**
Uhhhhhhhhh. I guess no?

**Describe the feature you'd like**
Use a consistent way of selecting things from a list of options.

It's confusing when some features (like in the good example) use a listing where you type the number of the object in the list you want to interact with but others use a listing where you have to enter the name of the listing.
It should only require very little change to find a way to make it consistent.

(this is of course if you weren't planning to replace it with dropdowns (probably a good idea), buttons or slash commands anyways)

Good Example:
![image](https://user-images.githubusercontent.com/79610706/126854732-f466d630-b816-4e5e-90fb-a1124d49b623.png)

Bad Example:
![2021-07-24_04-15](https://user-images.githubusercontent.com/79610706/126854705-4b2011fd-5a94-4161-bff5-0610b9da4761.png)

**Comment by IanMitchell** [comment]
Yeah, this is very fair. Probably gonna wait for selects / slash commands to ship in v13 and switch to those tho 👍
